changelog.md

```diff
@@ -1,3 +1,96 @@
+<a name="6.1.0"></a>
+# 6.1.0 (2020-06-25)
+[Full Changelog](https://github.com/GoogleChrome/lighthouse/compare/v6.0.0...v6.1.0)
+
+We expect this release to ship to DevTools in [Chrome 85](https://chromiumdash.appspot.com/schedule), and to PageSpeed Insights within 2 weeks.
+
+## New Contributors
+
+Thanks to our new contributors 👽🐷🐰🐯🐻! 
+Loftie Ellis @lpellis
+Marvin Frachet @mfrachet 
+Matt Hobbs @Nooshu 
+Peter Marshall @psmarshall 
+
+## Notable Changes
+
+* If a page has publicly-accessible JavaScript source maps, Lighthouse will collect them to enhance the `unused-javascript` audit. In future versions of Lighthouse, source maps will be used for entirely new audits [#10990](https://github.com/GoogleChrome/lighthouse/pull/10990)
+* The report now uses `KiB` instead of `KB`. This is simply a label change; the value was and still is equal to `1024 bytes` [#10870](https://github.com/GoogleChrome/lighthouse/pull/10870)
+
+<img src="https://i.imgur.com/1gSdPw5.png" alt="Unused JavaScript with source maps" width="800px">
```

Comment:
should be with the second bullet point?

Reply:
I thought it was a nice showcase of both at the same time :)

Reply:
it shows both things tho

Reply:
> I thought it was a nice showcase of both at the same time :)

> it shows both things tho

oh, haha, fair enough